gpii/node_modules/flowManager/src/userKeyaction.js

```diff
@@ -0,0 +1,250 @@
+/*
+ * GPII User Login Component
```

Comment:
I think a name like userLoginChange or userLogonStateChange is more descriptive - there's nothing about the "key" which is acting